data/campaigns/Northern_Rebirth/scenarios/01_Breaking_the_Chains.cfg

```diff
@@ -427,15 +427,39 @@
             id="Al'Tar"
         [/filter]
 
-        [message]
-            speaker=unit
-            message= _ "What the...? (<i>Gurgle</i>) Killed... by... slaves?"
-        [/message]
-
-        [message]
-            speaker=second_unit
-            message= _ "(<i>Stab</i>)"
-        [/message]
+        [switch]
+            variable=second_unit.race
+            [case]
+                value="human"
+                [message]
+                    speaker=unit
+                    message= _ "What the...? (<i>Gurgle</i>) Killed... by... slaves?"
+                [/message]
+                [message]
+                    speaker=second_unit
+                    message= _ "(<i>Stab</i>)"
+                [/message]
+            [/case]
+            [case]
+                value="orc"
+                [message]
+                    speaker=unit
+                    #po: "Lesser" meaning of inferior rank or weaker.
+                    message= _ "What the...? (<i>Gurgle</i>) Defeated... by... a lesser orc?"
```

Comment:
Should use a less clunky term for lesser orc, say just a "grunt" or "whelp" or something of the like.